openapi/v1/fileCurator.yaml

```diff
@@ -118,6 +118,13 @@ paths:
           required: true
           schema:
             type: string
+        - description: Include file contents in the response.
```

Comment:
I thought the description could be more informative. E.g. 
`Set to true in order to include file contents, e.g. .h5, .zip` to the response`. Please double check with products.